msal/src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -60,8 +60,6 @@ internal class InteractiveRequest : RequestBase
             UIBehavior UIBehavior, IWebUI webUI)
             : base(authenticationRequestParameters)
         {
-            PlatformPlugin.PlatformInformation.ValidateRedirectUri(authenticationRequestParameters.RedirectUri,
-                authenticationRequestParameters.RequestContext);
```

Comment:
was this supposed to only be the case for embedded webview?

Reply:
This only works on embedded webview, not system browser

Reply:
shouldn't it then still be enabled for the place where it makes sense?

---
In reply to: [199665688](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1107#discussion_r199665688) [](ancestors = 199665688)